src/main/scripts/src/app/shared/perspectives/icon-picker/icon-picker.component.ts

```diff
@@ -44,132 +43,36 @@ import {LocalService} from '../../../core/local.service';
     ])
   ]
 })
-export class IconPickerComponent implements OnInit, OnDestroy {
+export class IconPickerComponent {
+
+  @Input()
+  public enabled: boolean;
```

Comment:
This way the component can handle animations for itself. (Without it, the instance *ngIf changes to false, the component dissapears without closing animation.